TCAT/Views/RouteSelectionView.swift

```diff
@@ -18,7 +18,7 @@ protocol RouteSelectionViewDelegate: class {
 class RouteSelectionView: UIView {
 
     private weak var delegate: RouteSelectionViewDelegate?
-
+        
```

Comment:
theres still whitespace here :O

Reply:
All throughout the app there's whitespace between delegate declarations and other variable declarations. This white space was there since before I did any edits (I had inserted a variable and removed it so Github marked an inexistent change; hence the +- whitespace)